boltdesignsystem/bolt
# Move Brightcove JS out of Twig footer template

## Jira

n/a

## Summary

Move Brightcove JS out of Twig footer template.

## Details

Move Brightcove JS into separate file to:
1. Clean up footer template
2. Make Brightcove JS easier to work with
3. Allow eslint to run on this code

This PR also moves a function out of a loop to fix an eslint error.

## How to test

- Review code changes
- Verify Brightcove videos still work as expected in PL. Try this one for example: `/pattern-lab/?p=components-card-replacement-with-video`
